package net.mv.expectedException;

import org.junit.Test;

public class ExpectedExceptionTest {

	@Test(expected = ArithmeticException.class)
	public void divisionWithException() {
		int i = 1/0;
	}
}
